web/pages/group/[...slugs]/index.tsx

```diff
@@ -196,85 +198,118 @@ export default function GroupPage(props: {
     {
       title: 'Leaderboards',
-      content: leaderboard,
+      content: leaderboardPage,
       href: groupPath(group.slug, 'leaderboards'),
+      key: 'leaderboards',
     },
     {
       title: 'About',
-      content: aboutTab,
+      content: aboutPage,
       href: groupPath(group.slug, 'about'),
+      key: 'about',
     },
   ]
 
-  const tabIndex = tabs
-    .map((t) => t.title.toLowerCase())
-    .indexOf(page ?? 'markets')
+  const pageContent = sidebarPages[sidebarIndex].content
+  const onSidebarClick = (key: string) => {
+    const index = sidebarPages.findIndex((t) => t.key === key)
+    setSidebarIndex(index)
+  }
+
+  const joinOrAddQuestionsButton = (
+    <JoinOrAddQuestionsButtons
+      group={group}
+      user={user}
+      isMember={!!isMember}
+    />
+  )
 
   return (
-    <Page>
-      <SEO
-        title={group.name}
-        description={`Created by ${creator.name}. ${group.about}`}
-        url={groupPath(group.slug)}
-      />
-      <Col className="relative px-3">
-        <Row className={'items-center justify-between gap-4'}>
-          <div className={'sm:mb-1'}>
-            <div
-              className={'line-clamp-1 my-2 text-2xl text-indigo-700 sm:my-3'}
-            >
-              {group.name}
-            </div>
-            <div className={'hidden sm:block'}>
-              <Linkify text={group.about} />
-            </div>
+    <>
+      <header className="sticky top-0 z-50 w-full pb-2 md:hidden lg:col-span-12">
+        <div className="flex items-center  border-b border-gray-200 bg-white  px-4">
+          <div className="flex-shrink-0">
+            <Link href="/">
+              <a className="text-indigo-700 hover:text-gray-500 ">
+                <ArrowLeftIcon className="h-5 w-5" aria-hidden="true" />
+              </a>
+            </Link>
           </div>
-          <div className="mt-2">
-            <JoinOrAddQuestionsButtons
-              group={group}
-              user={user}
-              isMember={!!isMember}
-            />
+          <div className="ml-3">
+            <h1 className="text-lg font-medium text-indigo-700">
+              {group.name}
+            </h1>
           </div>
-        </Row>
-      </Col>
-      <Tabs
-        currentPageForAnalytics={groupPath(group.slug)}
-        className={'mx-2 mb-0 sm:mb-2'}
-        defaultIndex={tabIndex > 0 ? tabIndex : 0}
-        tabs={tabs}
-      />
-    </Page>
+        </div>
+      </header>
+      <div>
+        <div
+          className={
+            'mx-auto w-full pb-[58px] lg:grid lg:grid-cols-12 lg:gap-x-2 lg:pb-0 xl:max-w-7xl xl:gap-x-8'
+          }
+        >
+          <Toaster />
+          <GroupSidebar
+            groupName={group.name}
+            className="sticky top-0 hidden divide-gray-300 self-start pl-2 lg:col-span-2 lg:flex"
+            onClick={onSidebarClick}
+            joinOrAddQuestionsButton={joinOrAddQuestionsButton}
+            currentKey={sidebarPages[sidebarIndex].key}
```

Comment:
This should probably be represented in the URL, I think. Also could allow you to go back and forth.

A long time ago, you could go to `/group/JamesGroup/leaderboard` and it would show the leaderboard, etc.

Actually, now that these are not tabs, I think it's even more expected that the distinct pages of markets, about, leaderboard are different URLs. You can either change this PR, or follow up with the change.

Reply:
You might need to use nextjs catchall routes: https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes

Reply:
Actually, it might be better to make each a separate page as a new file under `pages/group/`.

E.g. `pages/group/my-group/markets`. Would be much better organization